python311Packages.awkward: 2.4.10 -> 2.5.0

GaetanLepage opened the pull request:
## Description of changes

Changelog: https://github.com/scikit-hep/awkward/releases/tag/v2.5.0

cc @veprbl 



<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

GaetanLepage committed:
python311Packages.awkward: 2.4.10 -> 2.5.1

Diff: https://github.com/scikit-hep/awkward/compare/refs/tags/v2.4.10...v2.5.1

Changelog: https://github.com/scikit-hep/awkward/releases/tag/v2.5.1

natsukium commented:
Result of `nixpkgs-review pr 267964` at d4b30940 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>python310Packages.coffea</li>
    <li><a href="https://gist.github.com/natsukium/52cb27da0a4a4038e4814425d163d7dc">python310Packages.dask-awkward</a></li>
    <li>python311Packages.coffea</li>
    <li><a href="https://gist.github.com/natsukium/e108d93388862f4391b04460fc22dc76">python311Packages.dask-awkward</a></li>
  </ul>
</details>
<details>
  <summary>10 packages built successfully:</summary>
  <ul>
    <li>python310Packages.awkward</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.vector</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.vector</li>
  </ul>
</details>

GaetanLepage committed:
python311Packages.dask-awkward: 2023.11.2 -> 2023.12.2

Diff: https://github.com/dask-contrib/dask-awkward/compare/refs/tags/2023.11.2...2023.12.2

Changelog: https://github.com/dask-contrib/dask-awkward/releases/tag/2023.12.2

GaetanLepage committed:
python311Packages.distributed: 2023.10.0 -> 2023.12.0

Diff: https://github.com/dask/distributed/compare/refs/tags/2023.10.0...2023.12.0

Changelog: https://github.com/dask/distributed/blob/2023.12.0/docs/source/changelog.rst

GaetanLepage commented:
This PR breaks some `coffea` tests:
```
=========================== short test summary info ============================
FAILED tests/test_analysis_tools.py::test_packed_selection_nminusone_dak[True] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_nminusone_dak[False] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_cutflow_dak[True] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_cutflow_dak[False] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_nminusone_dak_uproot_only[True] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_nminusone_dak_uproot_only[False] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_cutflow_dak_uproot_only[True] - NotImplementedError: Cannot determine length of collection with unknown par...
FAILED tests/test_analysis_tools.py::test_packed_selection_cutflow_dak_uproot_only[False] - NotImplementedError: Cannot determine length of collection with unknown par...
```

GaetanLepage committed:
python311Packages.dask-yarn: fix build

natsukium commented:
Result of `nixpkgs-review pr 267964` at 1b8eba6f run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/natsukium/3c3d0c389c38f7494519adbda3475f14">python310Packages.coffea</a></li>
    <li><a href="https://gist.github.com/natsukium/447b2c5cd7b6ad366a86961939a711b4">python311Packages.coffea</a></li>
  </ul>
</details>
<details>
  <summary>38 packages built successfully:</summary>
  <ul>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>python310Packages.awkward</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.ome-zarr</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.tsfresh</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.vector</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.vector</li>
  </ul>
</details>

GaetanLepage committed:
python311Packages.coffea: 2023.10.0.rc1 -> 2023.12.0

https://github.com/CoffeaTeam/coffea/releases/tag/v2023.12.0

GaetanLepage committed:
python311Packages.dask: 2023.10.1 -> 2023.12.0

Diff: https://github.com/dask/dask/compare/refs/tags/2023.10.1...2023.12.0

Changelog: https://docs.dask.org/en/latest/changelog.html

GaetanLepage committed:
python311Packages.awkward-cpp: 26 -> 27

Changelog: https://github.com/scikit-hep/awkward/releases/tag/v2.5.1

GaetanLepage committed:
python311Packages.uproot: 5.1.2 -> 5.2.0

Changelog: https://github.com/scikit-hep/uproot5/releases/tag/v5.2.0

veprbl commented on pkgs/development/python-modules/coffea/default.nix:
```diff
@@ -31,20 +31,19 @@
 buildPythonPackage rec {
   pname = "coffea";
-  version = "2023.10.0.rc1";
+  version = "2023.12.0.rc1";
```
2023.12.0 (not rc!) has just released
https://github.com/CoffeaTeam/coffea/releases/tag/v2023.12.0

GaetanLepage replied:
Indeed ! I just added it.

GaetanLepage commented:
On paper, this PR is ready. However, as `jaxlib` is currently broken, I cannot test the packages updated here.
I suggest to hold it until `jaxlib` gets fixed.

GaetanLepage commented:
Result of `nixpkgs-review pr 267964` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.bpycv.dist</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit.dist</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.stytra.dist</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.sunpy.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
  </ul>
</details>
<details>
  <summary>80 packages failed to build:</summary>
  <ul>
    <li>deface</li>
    <li>deface.dist</li>
    <li>python310Packages.awkward</li>
    <li>python310Packages.awkward.dist</li>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.bsuite.dist</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.clifford.dist</li>
    <li>python310Packages.coffea</li>
    <li>python310Packages.coffea.dist</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.correctionlib.dist</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-glm.dist</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-ml.dist</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.datashader.dist</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.devito.dist</li>
    <li>python310Packages.insightface</li>
    <li>python310Packages.insightface.dist</li>
    <li>python310Packages.mmcv</li>
    <li>python310Packages.mmcv.dist</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.mplhep.dist</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.qiskit-machine-learning.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.sparse.dist</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stumpy.dist</li>
    <li>python310Packages.tsfresh</li>
    <li>python310Packages.tsfresh.dist</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.uproot.dist</li>
    <li>python310Packages.vector</li>
    <li>python310Packages.vector.dist</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.awkward.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.correctionlib.dist</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.mplhep.dist</li>
    <li>python311Packages.rlax</li>
    <li>python311Packages.rlax.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.uproot.dist</li>
    <li>python311Packages.vector</li>
    <li>python311Packages.vector.dist</li>
  </ul>
</details>
<details>
  <summary>242 packages built:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python310Packages.agate-sql</li>
    <li>python310Packages.agate-sql.dist</li>
    <li>python310Packages.albumentations</li>
    <li>python310Packages.albumentations.dist</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.aplpy.dist</li>
    <li>python310Packages.awkward-cpp</li>
    <li>python310Packages.awkward-cpp.dist</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.batchgenerators.dist</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.boxx.dist</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.casa-formats-io.dist</li>
    <li>python310Packages.cf-xarray</li>
    <li>python310Packages.cf-xarray.dist</li>
    <li>python310Packages.crate</li>
    <li>python310Packages.crate.dist</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway.dist</li>
    <li>python310Packages.dask-histogram</li>
    <li>python310Packages.dask-histogram.dist</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-image.dist</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-jobqueue.dist</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-mpi.dist</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.dask-yarn.dist</li>
    <li>python310Packages.dask.dist</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.distributed.dist</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dremel3dpy.dist</li>
    <li>python310Packages.easyocr</li>
    <li>python310Packages.easyocr.dist</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.env-canada.dist</li>
    <li>python310Packages.glueviz</li>
    <li>python310Packages.glueviz.dist</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.glymur.dist</li>
    <li>python310Packages.gradio</li>
    <li>python310Packages.gradio.dist</li>
    <li>python310Packages.gymnasium</li>
    <li>python310Packages.gymnasium.dist</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.handout.dist</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imagecorruptions.dist</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imageio.dist</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.intake-parquet.dist</li>
    <li>python310Packages.intake.dist</li>
    <li>python310Packages.k-diffusion</li>
    <li>python310Packages.k-diffusion.dist</li>
    <li>python310Packages.lime</li>
    <li>python310Packages.lime.dist</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.moviepy.dist</li>
    <li>python310Packages.mung</li>
    <li>python310Packages.mung.dist</li>
    <li>python310Packages.muscima</li>
    <li>python310Packages.muscima.dist</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-console.dist</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.napari-svg.dist</li>
    <li>python310Packages.napari.dist</li>
    <li>python310Packages.ome-zarr</li>
    <li>python310Packages.ome-zarr.dist</li>
    <li>python310Packages.omrdatasettools</li>
    <li>python310Packages.omrdatasettools.dist</li>
    <li>python310Packages.open-clip-torch</li>
    <li>python310Packages.open-clip-torch.dist</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pims.dist</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.psd-tools.dist</li>
    <li>python310Packages.pvextractor</li>
    <li>python310Packages.pvextractor.dist</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyfftw.dist</li>
    <li>python310Packages.pyrender</li>
    <li>python310Packages.pyrender.dist</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.pyvista.dist</li>
    <li>python310Packages.qcodes</li>
    <li>python310Packages.qcodes-contrib-drivers</li>
    <li>python310Packages.qcodes-contrib-drivers.dist</li>
    <li>python310Packages.qcodes.dist</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.reproject.dist</li>
    <li>python310Packages.scikit-image</li>
    <li>python310Packages.scikit-image.dist</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.sfepy.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.slicedimage.dist</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.spectral-cube.dist</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.streamz.dist</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.test-tube.dist</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tifffile.dist</li>
    <li>python310Packages.webdataset</li>
    <li>python310Packages.webdataset.dist</li>
    <li>python310Packages.zcs</li>
    <li>python310Packages.zcs.dist</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.awkward-cpp</li>
    <li>python311Packages.awkward-cpp.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>

veprbl reviewed (approved):
LGTM

GaetanLepage commented:
Result of `nixpkgs-review pr 267964` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>30 packages marked as broken and skipped:</summary>
  <ul>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffe.bin</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.caffeWithCuda.bin</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.paddleocr</li>
    <li>python310Packages.paddleocr.dist</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit.dist</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.stytra.dist</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.sunpy.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python310Packages.unstructured-inference</li>
    <li>python310Packages.unstructured-inference.dist</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.bsuite.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.rlax</li>
    <li>python311Packages.rlax.dist</li>
  </ul>
</details>
<details>
  <summary>324 packages built:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>intensity-normalization (python311Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python311Packages.intensity-normalization.dist)</li>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python310Packages.agate-sql</li>
    <li>python310Packages.agate-sql.dist</li>
    <li>python310Packages.albumentations</li>
    <li>python310Packages.albumentations.dist</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.aplpy.dist</li>
    <li>python310Packages.awkward</li>
    <li>python310Packages.awkward-cpp</li>
    <li>python310Packages.awkward-cpp.dist</li>
    <li>python310Packages.awkward.dist</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.batchgenerators.dist</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.boxx.dist</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.bpycv.dist</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.casa-formats-io.dist</li>
    <li>python310Packages.cf-xarray</li>
    <li>python310Packages.cf-xarray.dist</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.clifford.dist</li>
    <li>python310Packages.coffea</li>
    <li>python310Packages.coffea.dist</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.correctionlib.dist</li>
    <li>python310Packages.crate</li>
    <li>python310Packages.crate.dist</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway.dist</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-glm.dist</li>
    <li>python310Packages.dask-histogram</li>
    <li>python310Packages.dask-histogram.dist</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-image.dist</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-jobqueue.dist</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-ml.dist</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-mpi.dist</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.dask-yarn.dist</li>
    <li>python310Packages.dask.dist</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.datashader.dist</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.devito.dist</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.distributed.dist</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dremel3dpy.dist</li>
    <li>python310Packages.easyocr</li>
    <li>python310Packages.easyocr.dist</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.env-canada.dist</li>
    <li>python310Packages.glueviz</li>
    <li>python310Packages.glueviz.dist</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.glymur.dist</li>
    <li>python310Packages.gradio</li>
    <li>python310Packages.gradio.dist</li>
    <li>python310Packages.gymnasium</li>
    <li>python310Packages.gymnasium.dist</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.handout.dist</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imagecorruptions.dist</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imageio.dist</li>
    <li>python310Packages.insightface</li>
    <li>python310Packages.insightface.dist</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.intake-parquet.dist</li>
    <li>python310Packages.intake.dist</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.intensity-normalization.dist</li>
    <li>python310Packages.k-diffusion</li>
    <li>python310Packages.k-diffusion.dist</li>
    <li>python310Packages.lime</li>
    <li>python310Packages.lime.dist</li>
    <li>python310Packages.mmcv</li>
    <li>python310Packages.mmcv.dist</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.moviepy.dist</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.mplhep.dist</li>
    <li>python310Packages.mung</li>
    <li>python310Packages.mung.dist</li>
    <li>python310Packages.muscima</li>
    <li>python310Packages.muscima.dist</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-console.dist</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.napari-svg.dist</li>
    <li>python310Packages.napari.dist</li>
    <li>python310Packages.ome-zarr</li>
    <li>python310Packages.ome-zarr.dist</li>
    <li>python310Packages.omrdatasettools</li>
    <li>python310Packages.omrdatasettools.dist</li>
    <li>python310Packages.open-clip-torch</li>
    <li>python310Packages.open-clip-torch.dist</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pims.dist</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.psd-tools.dist</li>
    <li>python310Packages.pvextractor</li>
    <li>python310Packages.pvextractor.dist</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyfftw.dist</li>
    <li>python310Packages.pyrender</li>
    <li>python310Packages.pyrender.dist</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.pyvista.dist</li>
    <li>python310Packages.qcodes</li>
    <li>python310Packages.qcodes-contrib-drivers</li>
    <li>python310Packages.qcodes-contrib-drivers.dist</li>
    <li>python310Packages.qcodes.dist</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.qiskit-machine-learning.dist</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.reproject.dist</li>
    <li>python310Packages.scikit-image</li>
    <li>python310Packages.scikit-image.dist</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.sfepy.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.slicedimage.dist</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.sparse.dist</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.spectral-cube.dist</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.streamz.dist</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stumpy.dist</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.test-tube.dist</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tifffile.dist</li>
    <li>python310Packages.tsfresh</li>
    <li>python310Packages.tsfresh.dist</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.uproot.dist</li>
    <li>python310Packages.vector</li>
    <li>python310Packages.vector.dist</li>
    <li>python310Packages.webdataset</li>
    <li>python310Packages.webdataset.dist</li>
    <li>python310Packages.zcs</li>
    <li>python310Packages.zcs.dist</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.awkward-cpp</li>
    <li>python311Packages.awkward-cpp.dist</li>
    <li>python311Packages.awkward.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.correctionlib.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.mplhep.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.uproot.dist</li>
    <li>python311Packages.vector</li>
    <li>python311Packages.vector.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
